packages/plugin-0g/src/actions/upload.ts

```diff
@@ -0,0 +1,185 @@
+import {
+    Action,
+    HandlerCallback,
+    IAgentRuntime,
+    Memory,
+    State,
+    ModelClass,
+    Content,
+    ActionExample
+} from "@ai16z/eliza";
+import { Indexer, ZgFile, getFlowContract } from '@0glabs/0g-ts-sdk';
+import { ethers } from 'ethers';
+import { composeContext } from "@ai16z/eliza";
+import { generateObject } from "@ai16z/eliza";
+import { promises as fs } from 'fs';
+
+
+const uploadTemplate = `Respond with a JSON markdown block containing only the extracted values. Use null for any values that cannot be determined.
```

Comment:
Can we move this to a templates file for maintainability

Reply:
Sure, thanks, I've been fixed.